var/spack/repos/builtin/packages/migraphx/package.py

```diff
@@ -85,9 +87,20 @@ class Migraphx(CMakePackage):
         depends_on(f"rocblas@{ver}", when=f"@{ver}")
         depends_on(f"miopen-hip@{ver}", when=f"@{ver}")
 
-    for ver in ["5.7.0", "5.7.1", "6.0.0", "6.0.2"]:
+    for ver in ["5.7.0", "5.7.1"]:
         depends_on(f"composable-kernel@{ver}", when=f"@{ver}")
 
+    for ck_version, ck_commit in [
+        ("6.0.0", "70eefcf4f263aa5c25f3c9ff0db8f6f199ef0fb9"),
+        ("6.0.2", "70eefcf4f263aa5c25f3c9ff0db8f6f199ef0fb9"),
+    ]:
+        resource(
+            name="composable-kernel",
+            git="https://github.com/ROCm/composable_kernel.git",
+            commit=ck_commit,
+            when="@" + ck_version,
+        )
+
```

Comment:
Curious.  Why is this package now a resource instead of a dependency?

Reply:
@tldahlgren , Migraphx 6.0 is not built with 6.0 tag of composable-kernel. 
It requires the commit **70eefcf4f263aa5c25f3c9ff0db8f6f199ef0fb9** from the composable-kernel.
Please refer the below update came in as part of migraphx 6.0.2 release.
https://github.com/ROCm/AMDMIGraphX/commit/05e3659838a1571ec863fbe9181ba70adc6b8ccc

Reply:
@tldahlgren ,
Updated the recipe with couple of upcoming changes in migraphx so that composable kernel dependency with the specified commit does not exist.